src/chuchichaestli/metrics/backbones/inception.py

```diff
@@ -0,0 +1,433 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision
+import timm
+import yaml
+import os
+
+try:
+    from torchvision.models.utils import load_state_dict_from_url
+except ImportError:
+    from torch.utils.model_zoo import load_url as load_state_dict_from_url
+from torchvision import transforms
+import torchvision.transforms.functional as TF
+
+
+from chuchichaestli.metrics.backbones.backbone_encoder import Encoder
+
+from chuchichaestli.metrics.backbones.resizer import pil_resize
+
+# Inception weights ported to Pytorch from
+# http://download.tensorflow.org/models/image/imagenet/inception-2015-12-05.tgz
+FID_WEIGHTS_URL = "https://github.com/mseitzer/pytorch-fid/releases/download/fid_weights/pt_inception-2015-12-05-6726825d.pth"  # noqa: E501
```

Comment:
See @bil-y comment on the pretrained weights URLs: https://github.com/CAIIVS/chuchichaestli/pull/87/files#r1888447482